Index.html

```diff
@@ -52,10 +52,182 @@ <h1>
+<div class="work-description">
+    <h1 class="work-title">Multi-Post <br> Stories</h1>
+    <ul class="job-description">
+        <li>CANOPY</li>
+        <li>Back End Dev</li>
+        <li>2015</li>
+    </ul>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="work-technologie">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+    </ul>
+    <button class="button-btn">See Project</button>
+</div>
+</article>
+</section>
+
+<section id="portfolio">
+    <article class="portfolio-work">
+        <div>
+            <h1>About <br> Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+         </div>
+         <div>
+            <a href="" class="connect">
+                Let's Connect
+            </a>
+            <ul class="work-technologie">
+                <li>
+                    <a href="">
+                        <img src="./assets/Github icon.svg" alt="Github">
+                    </a>
+                </li>
+                    <li class="icons">
+                    <a href="">
+                        <img src="./assets/Linkedin icon.svg" alt="Linkedin">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./assets/Angelist icon.svg" alt="Angelist">
+                    </a>
+                </li>
+                <li class="icons">
+                    <a href="">
+                        <img src="./assets/Twitter icon.svg" alt="Twitter">
+                    </a>
+                </li>
+                <li class="icons">
+                    <a href="">
+                        <img src="./assets/Medium icon.svg" alt="Medium">
+                    </a>
+                </li>
+            </ul>
+        <div class="button">Get My Resume</div>
+         </div>
+    </article>
+    <article>
+        <div>
+            <h1>Languages</h1>
+        </div>
+        <div class="language-layout">
+            <div class="language-content">
+                <img src="./assets/javascript circle.svg" alt="">
+                <label class="pl-">js</label>
+            </div>
+            <div class="language-content">
+                <img src="./assets/html circle.svg" alt="">
+                <label class="pl-">html</label>
+            </div>
+            <div class="language-content">
+                <img src="./assets/css circle.svg" alt="">
+                <label class="pl-">css</label>
+            </div>
+        </div>
+        <div>
+            <h1 class="bottom-line">Frameworks</h1>
+        </div>
+        
+        <div>
+            <h1 class="bottom-line">Skills</h1>
+        </div>
+    </article>
```

Comment:
- Kindly, The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`👌
and also technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li` or semantic reasons.👌

Reply:
busy with it